kit/golden-layout/styles.scss

```diff
@@ -23,11 +24,23 @@
       }
     }
 
+    .lm_tabs {
+      flex: 1;
+      position: relative;
+      overflow: hidden;
+      margin-bottom: -1px; // Allows vertical overflow of 1px, without scrollbars
+    }
+
+    .lm_controls {
+      position: relative;
+    }
+
     .lm_tab {
       display: flex;
       align-items: center;
       margin: 2px 0 0 2px;
       padding: 4px 8px;
+      max-width: calc(100% - 25px);
```

Comment:
Is this 25px the width of the add button?  Maybe pop it into a SASS variable to self-document it in the two places we are using.  Should we also set on the button itself to make explicit?

Reply:
Kind of - it's whats needed to allow for the add button, but its actually that width plus a bit of margin to balance out the right padding added to the header by GL.

Still, will put it in a var to help document it a bit